src/PrestaShopBundle/Resources/config/services/adapter/data_configuration.yml

```diff
@@ -216,3 +216,11 @@ services:
             - '@prestashop.adapter.data_provider.language'
             - '@prestashop.adapter.cache_clearer'
             - '@=service("prestashop.adapter.legacy.configuration").get("_PS_TRANSLATIONS_DIR_")'
+
+    prestashop.adapter.webservice.webservice_account_eraser:
```

Comment:
how it is a data configuration? i think this service should be declared under `webservice.yml`.